packages/tangle/scheduler.go

```diff
@@ -251,22 +252,13 @@ func (s *Scheduler) submit(message *Message) error {
 		return ErrNotRunning
 	}
 
-	nodeID := identity.NewID(message.IssuerPublicKey())
-	nodeMana := s.tangle.Options.SchedulerParams.AccessManaRetrieveFunc(nodeID)
+	// TODO: check if nodes have MinMana here
```

Comment:
We don't want to do it right now, because it would make the network inaccessible for the nodes without mana, so I removed this check to allow them access and added the TODO to mark the place where the check needs to be added in the undefined future. Creating an issue for this would introduce unnecessary clutter I think (or maybe not if we put it in the Freezebox flow?). What do you think?

Reply:
The idea of creating an issue is that we won't forget about it and that it can be prioritized.

If you think that we won't foget about it and that it will be prioritized w.o an issue than it isn't neccessary :-)

The reason I am writing this comment is that in my opinion TODOs should be a good tool to uses while working on a PR. But for prioritizing later work we have github.

Reply:
I totally agree! It's not something that needs to be done for this feature to work. It's more to point someone in the future where he should add the check instead of figuring it out from scratch. I will change the working of this comment and remove the TODO, as it's not really a TODO :)